weaver/processes/wps_workflow.py

```diff
@@ -270,6 +271,13 @@ def collect_output(self,
                         try:
                             prefix = fs_access.glob(outdir)
                             key = cmp_to_key(cast(Callable[[Text, Text], int], locale.strcoll))
+
+                            # In case of stdout.log or stderr.log file not created
+                            if glob in ("stdout.log", "stderr.log"):
```

Comment:
we should use the literal string but rather the entries in the cwl package as this can have any value

Reply:
You mean using value of `glob`? 
If so then I would suggest to limit ourselves to `stdout.log` and `stderr.log` in order to avoid creating empty file of a glob that should not have an output, in order to raise consequently.

Reply:
What I mean is that it shouldn't be hardcoded `"stdout.log"` but `package["stdout"]` or similar to get the real file name placed in the CWL definition since this could be modified at any point.

Reply:
Good point, updated it, let me know if you're alright with the change